ws/mod.ts

```diff
@@ -36,7 +36,7 @@ export interface WebSocketCloseEvent {
 export function isWebSocketCloseEvent(
   a: WebSocketEvent,
 ): a is WebSocketCloseEvent {
-  return hasOwnProperty(a, "code");
+  return hasOwn(a, "code");
```

Comment:
How about `typeof a !== "string" && hasOwn(a, "code")`?

Reply:
the previous code works the same as the new
```
 Object.prototype.hasOwnProperty.call("a", "b")
 false
 Object.hasOwn("a", "b")
 false
```

if it is a string it will return false like before

Reply:
If think typescript is bugged in this scenario

Reply:
typescript seems supposing the first arg is object (not string) https://github.com/denoland/deno/blob/b518f5e1ba4c3281d4aee4c637f410db382ce751/cli/dts/lib.esnext.object.d.ts

Other solution might be type casting like `a as object` with `// deno-lint-ignore ban-types` lint directive

Reply:
Is it good now ?

Reply:
It should be something like the below, I guess

```ts
// deno-lint-ignore ban-types
return hasOwn(a as object, "code");
```

You can check it with `deno run ws/mod.ts`.

Reply:
I have other unrelated to this PR errors when I do this